stylesheets/styles.css

```diff
@@ -335,11 +335,13 @@ td {
   margin-right: 1.5em;
 }
 
-.horizontal {
-  float: center;
-  padding: .5em;
-  list-style: none;
-  font-size: 1.15em;
+.horizontal {/* replaced float:center with appropriate values*/
+padding: .5em;
```

Comment:
@sara-02 I think float:left would work in this case. What do you think?

Reply:
@shreyateeza  , i think float:left will not work..

Reply:
@AnkitKhatri55555, @shreyateeza the only way to find out is to run the changes locally and see how it is behaving.

Reply:
If `float:left` can do the job, then no need to add new attributes.

Reply:
@sara-02 , changing float:center to float:left results in this:
![screen shot 1939-12-06 at 6 10 24 pm](https://user-images.githubusercontent.com/36541665/36641561-6ea63a50-1a57-11e8-9f0d-ae7711a79640.png)